### PR TITLE
Fixes #373 Added Tooltips to the new activity, milestone, task, and requirements modals.

### LINKED ISSUE #373: On-Screen Help Instructions
While having a detailed user manual is helpful, I see no reason not to have on-screen instructions to guide the user in using the features presented to them. There is a lot of empty real estate within the UI and navigating through the program is not as intuitive as maybe it should be. With additional text instructions, the user can navigate through the menus and learn what the difference between a "task" and an "activity" is. The goal would be to eventually implement on-screen prompts for new users.

### PR DESCRIPTION
**Pull request by westfall123**
Tooltips were added to the milestone, activity, task, and requirements
modals to give the user more instruction and help guide them through 
the applicaiton.

Resolves: #373

### CODE REVIEW

**Comment by westfall123** [comment]
Yes, we used scene builder for the fxml and the information.png file was downloaded off a free icon webpage (https://www.flaticon.com/).

**Comment by rsanchez-wsu** [comment]
OK.  You will need to relocate the image to `src/edu/wright/cs/raiderplanner/content` and then ensure that an appropriate credit is given in `IMAGE-CREDITS.md`.  Note that since it is not an original work, the credit should include the URL where you obtained the image.

File: src/edu/wright/cs/raiderplanner/content/IMAGE-CREDITS.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ settings_about.png - GIMP image, created by Clayton D. Terrill
 settings_account.png - GIMP image, created by Clayton D. Terrill
 settings_general.png - GIMP image, created by Clayton D. Terrill
 settings_goBack.png - GIMP image, created by Clayton D. Terrill
-settings_theme.png - GIMP image, created by Clayton D. Terrill
\ No newline at end of file
+settings_theme.png - GIMP image, created by Clayton D. Terrill, created by Hayden M. Westfall information.png - https://www.flaticon.com/
```
Comment (rsanchez-wsu) [inline src/edu/wright/cs/raiderplanner/content/IMAGE-CREDITS.md]: This change does not make sense.  You should have `information.png` on a new line and then just the (complete) URL, like this:
```
information.png - https://www.example.com/path/to/some/image.png
```
Please fix this.

**Comment by rsanchez-wsu** [comment]
@westfall123 One additional thing, can you provide a screenshot showing the placement of the new information.png image in the application?

**Comment by rsanchez-wsu** [comment]
This has been replaced by #464

**Pull request closed without merging**